### PR TITLE
feat(provider/kubernetes): split kubernetes pod logs by container #3360

### PR DESCRIPTION
**Pull request by maggieneterval**
Closes [issue #3360](https://github.com/spinnaker/spinnaker/issues/3360)

Updates kubernetes instance logs and job logs to be divided into tabs by container. Adds refresh functionality to instance logs:

spinnaker-k8s-log-screengrab

cc @danielpeach @lwander

### CODE REVIEW

**Comment by duftler** [comment]
Probably want to inline the screengrab so it is publicly visible.

**Comment by lwander** [comment]
Nice, big fan of the refresh & scroll to end additions as well.

**Comment by maggieneterval** [comment]
@lwander Can't take credit for the scroll to end button as that was already there, but thank you!

File: app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/react/JobManifestPodLogs.tsx [hunk 1]
```diff
@@ -11,17 +18,21 @@ export interface IJobManifestPodLogsProps {
...
 export interface IJobManifestPodLogsState {
+  containerLogs: IKubernetesContainerLog[];
   showModal: boolean;
-  output: string;
+  selectedContainerLog: IKubernetesContainerLog;
+  errorMessage: string;
 }
 
 // JobManifestPodLogs exposes pod logs for Job type manifests in the deploy manifest stage
 export class JobManifestPodLogs extends React.Component<IJobManifestPodLogsProps, IJobManifestPodLogsState> {
```
[the remark this answers is not in the document]
Comment (sbwsg) [reply app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/react/JobManifestPodLogs.tsx]: Yeah, I think that was the original intention: https://github.com/spinnaker/deck/pull/5537

The other logs modal is `app/scripts/modules/core/src/instance/details/console/consoleOutput.modal.controller.js` and accompanying template.

I've created a separate GH issue for de-duplicating: https://github.com/spinnaker/spinnaker/issues/3420

**Comment by maggieneterval** [comment]
@danielpeach thank you for the code review, just updated based on your comments!

File: app/scripts/modules/core/src/instance/details/console/consoleOutputLink.directive.js [hunk 2]
```diff
@@ -17,17 +19,20 @@ module.exports = angular
       bindToController: {
         instance: '=',
         text: '=?',
+        usesMultiOutput: '=?',
```
Comment (danielpeach) [inline app/scripts/modules/core/src/instance/details/console/consoleOutputLink.directive.js]: Can be `<?`, but not important.

**Comment by romil-punetha** [comment]
How can i see these logs? I've been deploying helm charts, yet i dont see any logs for my containers

**Comment by romil-punetha** [comment]
How can i see these logs? I've been deploying helm charts, yet i dont see any logs for my containers

**Pull request merged**